tracer/tracer.go

```diff
@@ -110,6 +126,20 @@ func HandleSendTracesResponse(resp *http.Response, err error) {
 
 func (tracer *epsagonTracer) getTraceReader() (io.Reader, error) {
 	version := "go " + runtime.Version()
+
+	for _, event := range tracer.events {
+		if event.Origin == "runner" {
+			jsonString, err := json.Marshal(tracer.labels)
```

Comment:
should break the loop after the runner event has found (alternatively - add some getRunnerEvent func)